backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmFromExternalUrlCommand.java

```diff
@@ -152,9 +153,9 @@ private ImportVmFromExternalProviderParameters buildImportVmParameters(VM vm) {
                 // in kvm we just copy the image, in other modes such as vmware or xen we use
                 // virt-v2v which converts the image format as well
                 if (vm.getOrigin() != OriginType.KVM) {
-                    disk.setVolumeFormat(getDiskVolumeFormat(
-                            disk.getVolumeType(),
-                            getStorageDomain().getStorageType()));
+                    disk.setVolumeFormat(disk.getBackup() == DiskBackup.Incremental ? VolumeFormat.COW :
```

Comment:
I don't think it's needed because for OVA we "know" that the data is valid (if incremental-backup is set, the format is qcow) and for other sources we won't get a disk with incremental-backup enabled but it doesn't seem harmful either

Reply:
ah no, that comment is actually wrong - we may import the vm/template to a storage domain of a different type..